### ISSUE TITLE
Landing page responsiveness on small screens

### ISSUE THREAD

**Issue by kieranjol** [finoradin/mattersinmediart.org#32]
Hi! Looking forward to properly viewing the site,looks like a wonderful resource. 
The homepage does not format correctly on my phone. I'm using Android 5.0/chrome/moto g 3rd edition/1280*720. The text is cut off in the left hand side.
The issue is with landscape and portrait mode. Two screenshots attached in both modes.
It seems,from just skimming through the site, that the homepage is mainly affected.

Edit: This also occurs with Firefox 46.0 on Ubuntu 14.04 when resizing the browser window.
![screenshot_20160515-224832](https://cloud.githubusercontent.com/assets/1127102/15276992/75715d9c-1aef-11e6-9b31-802e06ab0a31.png)
![screenshot_20160515-224807](https://cloud.githubusercontent.com/assets/1127102/15276997/808a5f94-1aef-11e6-80ce-182a9bf96008.png)

**Comment by kieranjol** [comment]
Footer of digital preservation page has an issue in landscape mode only.

![screenshot_20160515-225747](https://cloud.githubusercontent.com/assets/1127102/15277033/9c86736c-1af0-11e6-97c3-da8ed694572a.png)

**Comment by kieranjol** [comment]
Home page menu looks broken when refreshed in landscape mode.
![screenshot_20160515-230248](https://cloud.githubusercontent.com/assets/1127102/15277065/4735874e-1af1-11e6-89d0-c57bc25ca2f3.png)

**Comment by kieranjol** [comment]
In the digital preservation page, it seems like more headings should appear on the left?
http://mattersinmediaart.org/sustaining-your-collection.html

![screenshot_20160515-230055](https://cloud.githubusercontent.com/assets/1127102/15277076/84cbf980-1af1-11e6-8669-cc5a45c62f8f.png)

**Comment by finoradin** [comment]
Hi @kieranjol – thanks for taking the time to give us these great screenshots! You have filed several different and unique issues under this one thread. Could you please break them out into individual issues? While these are all related to rendering issues on your particular screen size, they will likely all be solved individually / one by one. Having discrete issues allows for us to have a more exacting dialog about the issues on an individual basis, and will allow us to close them one by one as they get fixed. Thanks!!

**Comment by kieranjol** [comment]
Absolutely, I wasn't sure if they were all unique enough for separate issues. 
Edit:
All are split up now into separate issues.

**Comment by kieranjol** [comment]
This also occurs with Firefox 46.0 on Ubuntu 14.04 when resizing the browser window.

**Comment by finoradin** [comment]
@kieranjol yeah - the site is responsive, so the issues you have filed aren't specific to your browser or device, they're related to screen size.
